.github/workflows/bump-version.py

```
@@ -2,13 +2,11 @@
 import sys
 
 sys.path.append("../../")
-from clang_tools import release_tag  # noqa E402
+from clang_tools import binary_repo, binary_tag  # noqa E402
 
 
 def get_latest_tag() -> str:
-    response = requests.get(
-        "https://api.github.com/repos/cpp-linter/clang-tools-static-binaries/releases/latest"
-    )
+    response = requests.get(f"{binary_repo}/releases/latest")
```

Ruff (0.12.2)

Probable use of `requests` call without timeout

(S113)